packages/console/src/components/CreateConnectorForm/Footer/index.tsx

```diff
@@ -71,6 +71,7 @@ function Footer({
         <QuotaGuardFooter>
           <Trans
             components={{
+              b: <b />,
```

Comment:
nit: If I remember correctly, the i18next can render `<strong />` properly without writing this `<Trans />` rules. I've been using this on the official website

Reply:
let me take a look and update to `<strong>`

Reply:
works